### DISCUSSION TITLE
Thank you for all your hard work

### DISCUSSION THREAD

**Discussion by danwritecode** [Diizzayy/nuxt-graphql-client#381] [General]
Hey All, I just wanted to say what an incredible library this is. It's so simple to use and I haven't run into any bugs thus far. This has really made graphql a joy to use from the automatic type gen to the querying syntax with the composable. 

Thank you!

**Comment by Diizzayy** [comment]
@danwritecode Thank you for the kind words! I'm Incredibly happy to hear that you've enjoyed using this package.
